source/resource/qx/tool/schema/compile-1-0-0.json

```diff
@@ -247,29 +268,34 @@
             "type": ["boolean","string"],
             "description": "When true, output the typescript definition into ``./compiled/source/qooxdoo.d.ts`. If you use a string instead of true, the string is the name of the file which is generated inside the target output directory"
           },
+          "mangle-privates": {
+            "description": " Whether and how to mangle private identifiers. Can be overridden on a per application basis. Can be overridden on a per application basis.",
```

Comment:
Repeated sentence "Can be overridden on a per application basis"

Also, it should be "per-application" (with a dash) here and other places where that phrase is used